initfiles/examples/embed/couchbase-simple.ecl

```diff
@@ -0,0 +1,112 @@
+IMPORT couchbase;
+/******************************************************************************************
+   This set of example queries are based on the couchbase sample bucket 'travel-sample'
+   The sample data is easily set-up, pelase refer to the couchbase documentation for set-up
+*******************************************************************************************/
+
+server := '127.0.0.1';            //change this to point to your couchbase
+thebucket := 'travel-sample';
+
+namerec := RECORD
+    string name;
+END;
+
+travelrec := RECORD
+    integer id;
+    string name;
+    string callsign;
+    string country;
+    string5 iata;
+    string5 icao;
+    string type;
+END;
+
+georec := RECORD
+    integer alt;
+    real lat;
+    real lon;
+END;
+
+airportrec := RECORD
+    integer id;
+    string airportname;
+    string city;
+    string country;
+    georec geo;
+    string5 faa;
+    string  tz;
+    string5 icao;
+    string type;
+END;
+
+/*
+ --Sample Airport record--
+ "airportname": "Calais Dunkerque",
+    "city": "Calais",
+    "country": "France",
+    "faa": "CQF",
+    "geo": {
+            "alt": 12,
+            "lat": 50.962097,
+            "lon": 1.954764
+    },
+    "icao": "LFAC",
+    "id": 1254,
+    "type": "airport",
+    "tz": "Europe/Paris"
+*/
+/*
+  --sample airline record--
+        "callsign": null,
+        "country": "United States",
+        "iata": "WQ",
+        "icao": "PQW",
+        "id": 13633,
+        "name": "PanAm World Airways",
+        "type": "airline"
+*/
+
+dataset(travelrec) fulltravelrecords() := EMBED(couchbase : server(server), bucket(thebucket), detailed_errcodes(1), operation_timeout(5.5), config_total_timeout(15))
+    select mybucketalias.* from `travel-sample` as mybucketalias limit 500;
+ENDEMBED;
+
+dataset(namerec) airlinenames() := EMBED(couchbase : server(server), bucket(thebucket), detailed_errcodes(1), operation_timeout(5.5), config_total_timeout(15))
```

Comment:
I'm only interested in the set of strings resulting from this query, if I declare string return, it makes it a scalar result. If I use travelrec, I'll have a lot of superfluous fields, what's the right way to do this?